.eslintrc.yml

```diff
@@ -10,6 +10,7 @@ rules:
   prettier/prettier: error
   block-scoped-var: error
   eqeqeq: error
+  node/no-unpublished-require: off
```

Comment:
This is going to get blown away by the next update.  If you need something like this, please push it all the way down into the `samples` directory `.eslintrc.yml` file.

Reply:
OK... Then "npm run fix" will complain about : error  "@google-cloud/nodejs-repo-tools" is not published  node/no-unpublished-require

Reply:
I've run lint and it's not complaining. Thanks.